packages/integration_test/pubspec.yaml

```
@@ -15,8 +15,16 @@ dependencies:
   flutter_test:
     sdk: flutter
   path: ^1.6.4
+  test_core:
```

@jakemac53 @natebosch - LGTY?

---

This can be updated now

```suggestion
  test_core: 0.3.12-nullsafety.6
```

Unfortunately null safety leaves us in a state where we'll need to pin to a single version since we can't really pin to a feature version currently.

---

Thanks, I updated it but there's a conflict with flutter_test that is pinned to the previous version. I have opened https://github.com/flutter/flutter/pull/68771 to update that separately.